src/main/java/com/blackducksoftware/integration/hub/alert/digest/DigestRemovalProcessor.java

```diff
@@ -93,7 +93,7 @@ private boolean processPolicyNotifications(final Map<String, NotificationEntity>
     }
 
     private boolean processVulnerabilityNotifications(final String cacheKey, final Map<String, NotificationEntity> categoryMap, final NotificationEntity entity) {
-        final String notificationType = entity.getNotificationType();
+        final String notificationType = entity.getNotificationType().name();
```

Comment:
You can also try getting just the enum here as well but would require some additional changes to the maps below